src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -472,6 +472,34 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
   ============================================================
+                                      InjectTargetPathMetadata
+  
+  Update TargetPathWithTargetPlatformMoniker with target framework
+  identifier and version metadata.  This is so that the
+  ImplicitlyExpandNETStandardFacades target can determine if a
+  referenced project needs the .NET Standard facades even if
+  the project hasn't been compiled to disk yet.
+  
+  See https://github.com/dotnet/sdk/issues/1403 for more context
+  ============================================================
+  -->
+  
+  <Target Name="InjectTargetPathMetadata"
+        BeforeTargets="GetTargetPath"
+        DependsOnTargets="GetTargetPathWithTargetPlatformMoniker">
+
+    <ItemGroup>
+      <TargetPathWithTargetPlatformMoniker>
+        <TargetFrameworkIdentifier>$(TargetFrameworkIdentifier)</TargetFrameworkIdentifier>
+        <TargetFrameworkVersion>$(_TargetFrameworkVersionWithoutV)</TargetFrameworkVersion>
```

Comment:
There is a `TargetPlatformVersion` and unlike `TargetFrameworkVersion`, it [does not start with a "v"](https://github.com/Microsoft/msbuild/blob/68bf979f5ea8f842d26d9ce48580383edfa3e944/src/Tasks/Microsoft.Common.CurrentVersion.targets#L87).